plugin/trino-hive/src/main/java/io/trino/plugin/hive/s3select/S3SelectCsvRecordReader.java

```diff
@@ -91,4 +92,11 @@ public SelectObjectContentRequest buildSelectObjectRequest(Properties schema, St
 
         return selectObjectRequest;
     }
+
+    @Override
+    protected String getFieldDelimiter(Properties schema)
```

Comment:
It's kind of weird to override this method, if the only use place is in the `buildSelectObjectRequest` which is also overridden. 
Remove `io.trino.plugin.hive.s3select.S3SelectLineRecordReader#getFieldDelimiter`  implementation, as it's no longer used. Remove the definition, since it's redundant

Reply:
Makes sense, I've removed this method in the latest commit.